apps/meteor/client/views/invite/hooks/useSamlInviteToken.ts

```diff
@@ -0,0 +1,9 @@
+import { useSessionStorage } from '@rocket.chat/fuselage-hooks';
+import { useRouteParameter } from '@rocket.chat/ui-contexts';
+
+const KEY = 'saml_invite_token';
+
+export const useSamlInviteToken = () => {
+	const token = useRouteParameter('hash');
```

Comment:
I see you're setting the token here and inside InvitePage. I think you can choose one or the other, probably just here would be enough.

Reply:
Hmmm I'm not sure. As far as I checked in our code for `useSessionStorage`, the `initialValue` param (the second one, which I'm using in this file) is only assigned to the key if it hasn't been created yet. So in case the key already exists, I need to set it in the `InvitePage` again (e.g. logging out and trying to login again with an invite in the same tab would not work)

Reply:
Oh okay then. Let's leave it like this